lib/aiko/led.py

```diff
@@ -2,9 +2,9 @@
 #
 # Usage
 # ~~~~~
-# import aiko.led
-# aiko.led.initialise()
-# aiko.led.pixel(aiko.led.red, 0, True)
+# from lib.aiko import led
+# led.initialise()
+# led.pixel(aiko.led.red, 0, True)
```

Comment:
Kinda fascinated… `aiko.led` is imported and initialised by `main.py`, so we could kinda eliminate those lines; what do we get from the switch from `aiko.led.pixel` to `led.pixel`?

Reply:
yes and no, I ran this code from repl where it was not visible. There are some weird visibility issues between what's visible from repl and from code run by main.py.
I've had a lot of issues with led.dim being a different value when set from MQTT, or from code. I'm still trying to figure out why, it's very perplexing.
If you are experienced with micropython namespace issues, let me know and I'll share more details.

Reply:
I'm not entirely clear on it either, it strikes me. I'd swear `aiko` is always imported on my REPL. Perhaps that's more to do the with limits of the soft reset inflicted by Control-D, though. (I've also noticed poor or zero reliability of MQTT hook-up after ^D, which I need to nut out.)

Reply:
Ok, clear now: we've got the double import problem (see PR-level convo [here](#issuecomment-769403945)) and when `main.py` runs it's doing so with the same globals as the REPL. So, straight after booting you can see `aiko`, `application`, `common`, `configuration`, `plugins`, and a bunch of others even though you didn't import them. ^D kicks it all off again.